comp/aggregator/demultiplexer/demultiplexerimpl/demultiplexer.go

```diff
@@ -106,3 +130,41 @@ func newDemultiplexer(deps dependencies) (provides, error) {
 func (demux demultiplexer) LazyGetSenderManager() (sender.SenderManager, error) {
 	return demux, nil
 }
+
+func (demux demultiplexerEndpoint) dumpDogstatsdContexts(w http.ResponseWriter, _ *http.Request) {
+	path, err := dumpDogstatsdContextsImpl(demux.Comp, demux.Config)
```

Comment:
Could we rename `dumpDogstatsdContextsImpl` to just `dumpDogstatsdContexts` I'm not sure `Impl` adds much value to the function name

Reply:
fixed in https://github.com/DataDog/datadog-agent/pull/25367/commits/52766c3db540726f5678fee0caadbb7016a139ea